### PR TITLE
Remove unused dependencies

### LINKED ISSUE #68: Remove unnecessary dependencies from package.json in API, Public, Admin
After the components are split, remove unnecessary package.json dependencies. E.g. API back-end shouldn't depend on Angular (while front-end will). E.g. remove nx.

### PR DESCRIPTION
**Pull request by basilv**
Fixes #68

### CODE REVIEW

**Comment by basilv** [comment]
work in progress, started with removing API dependencies.

**Comment by basilv** [comment]
closing and reopening to try to fix failing deployment verification check.

**Comment by basilv** [comment]
Ready for review. More could be done to simplify the code and reduce dependencies but this would require more impactful changes so deferred to a separate issue post-cutover.
API, libs, public, and admin all had dependencies reduced. Very minor changes to admin and public to remove two dependencies, otherwise other dependencies were completely unused.

File: admin/package.json [hunk 1]
```diff
@@ -25,24 +25,16 @@
...
-    "linkify-it": "^3.0.3",
     "lodash": "^4.17.21",
     "moment": "^2.29.2",
     "ngx-bootstrap": "^7.1.2",
     "ngx-dropzone": "^3.0.0",
-    "object-hash": "^3.0.0",
-    "popper.js": "^1.16.1",
     "remeda": "^0.0.32",
     "rxjs": "~7.5.5",
-    "tslib": "^2.3.1",
```
Comment (ianliuwk1019) [inline admin/package.json:44]: It build without tslib?

**Review by ianliuwk1019 — DISMISSED** [review]
Thanks Basil.

**Pull request merged**